src/components/ui/modal/modal.component.tsx

```diff
@@ -93,7 +94,15 @@ export class Modal extends React.PureComponent<ModalProps, State> {
     this.modalId = ModalService.hide(this.modalId);
   };
 
+  public forceRender = (): void => {
```

Comment:
```suggestion
  private forceRender = (): void => {
```

Should not be public, should it?
If not, it should be declared below lifecycle methods